contrib/vagrant/scripts/helpers.bash

```diff
@@ -106,7 +106,9 @@ function download_to {
     if [ ! -f "${cache_dir}/${component}" ]; then
         log "Downloading ${component}..."
 
-        ${WGET} -O "${cache_dir}/${component}" -nv "${url}"
+        ${WGET} -O "/tmp/${component}" -nv "${url}"
+        # Hide 'failed to preserve ownership' error
+        mv "/tmp/${component}" "${cache_dir}/${component}" 2>/dev/null
```

Comment:
What happens if a previous download was interrupted and not cleaned up from /tmp? As I understand, wget would automatically append `.1` to the component name, but then we would move the old version to the cache?

Should we clean (rm) the component if a file with that name is already present under /tmp, or alternatively use a timestamp for the temporary file (`DATE=$(date +%Y%m%d%H%M%S); ${WGET} -O "/tmp/${component}_${DATE}"`)? Or am I overthinking it?

Reply:
No, you're right, we need to clean `/tmp/`. Deleting the old component seems enough.